src/Routing/Reservoirs/Persistence_Level_Pool_Hybrid/module_persistence_levelpool_hybrid.F90

```diff
@@ -201,7 +201,7 @@ subroutine hybrid_init(this, water_elevation,  &
             ! Initialize level pool reservoir
             call this%state%levelpool_ptr%init(water_elevation, lake_area, &
             weir_elevation, weir_coeffecient, weir_length, dam_length, orifice_elevation, &
-            orifice_coefficient, orifice_area, lake_max_water_elevation, lake_number)
+            orifice_coefficient, orifice_area, lake_max_water_elevation, lake_number, 1)
```

Comment:
I'm fine either way but do we want to make this 1 and the 1 in `module_rfc_forecasts.F90`:159 a named constant to better indicated what is being passed?